sdk/nodejs/cmd/pulumi-language-nodejs/main.go

```diff
@@ -367,9 +367,16 @@ func (host *nodeLanguageHost) Run(ctx context.Context, req *pulumirpc.RunRequest
 		contract.IgnoreError(os.Stdout.Sync())
 		contract.IgnoreError(os.Stderr.Sync())
 		if exiterr, ok := err.(*exec.ExitError); ok {
-			// If the program ran, but exited with a non-zero error code.  This will happen often, since user
-			// errors will trigger this.  So, the error message should look as nice as possible.
+			// If the program ran, but exited with a non-zero error code.  This will happen often,
+			// since user errors will trigger this.  So, the error message should look as nice as
+			// possible.
 			if status, stok := exiterr.Sys().(syscall.WaitStatus); stok {
+				// 32 is the special exit code that means "we already logged all messages with the
+				// user, no need to report anything else.
+				if status.ExitStatus() == 32 {
```

Comment:
can we make this a constant so it's clear that 32 has some semantics here?

Reply:
sure :)